src/scss/components/card.scss

```
@@ -116,6 +116,34 @@
       object-fit: cover;
     }
 
+    .media-date {
+      position: relative;
+      color: #000;
+      background-color: #ffcd00;
+      text-align: center;
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      justify-content: center;
+      padding: 0.9rem 0.6rem 0.2rem 0.6rem;
+
+      .media-date--month {
```

```suggestion
      .media--date__month {
```
Should be underscores according to BEM since this is a part of a larger element.